### ISSUE TITLE
WSL2 Ubuntu package failure

### ISSUE THREAD

**Issue by VikingExplorer** [awslabs/aws-lambda-cpp#98]
Welcome to Ubuntu 20.04 LTS (GNU/Linux 4.19.104-microsoft-standard x86_64)

gunnar@Aspen:/mnt/c/Users/gunna$ cd /mnt/c/Dev/Couloir/Research/aws/HelloCMake
gunnar@Aspen:/mnt/c/Dev/Couloir/Research/aws/HelloCMake$ cd build
gunnar@Aspen:/mnt/c/Dev/Couloir/Research/aws/HelloCMake/build$ ls
gunnar@Aspen:/mnt/c/Dev/Couloir/Research/aws/HelloCMake/build$ cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=~/lambda-install
-- Configuring done
-- Generating done
-- Build files have been written to: /mnt/c/Dev/Couloir/Research/aws/HelloCMake/build
gunnar@Aspen:/mnt/c/Dev/Couloir/Research/aws/HelloCMake/build$ make
[100%] Built target demo
gunnar@Aspen:/mnt/c/Dev/Couloir/Research/aws/HelloCMake/build$ make aws-lambda-package-demo
[100%] Built target demo
make[3]: /mnt/c/Dev/Couloir/Research/aws/aws-lambda-cpp/build/packager: Command not found
make[3]: *** [CMakeFiles/aws-lambda-package-demo.dir/build.make:57: CMakeFiles/aws-lambda-package-demo] Error 127
make[2]: *** [CMakeFiles/Makefile2:77: CMakeFiles/aws-lambda-package-demo.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:84: CMakeFiles/aws-lambda-package-demo.dir/rule] Error 2
make: *** [Makefile:118: aws-lambda-package-demo] Error 2

**Comment by marcomagdy** [comment]
Unfortunately, I don't have a Windows machine with WSL2 available.

`/mnt/c/Dev/Couloir/Research/aws/aws-lambda-cpp/build/packager: Command not found`
Can you verify that the file exists and is executable?
Can you run it manually?

**Comment by VikingExplorer** [comment]
What would the exact command be?

**Comment by marcomagdy** [comment]
OK - I now see that the `packager` path is wrong.
It should be coming from your install directory `~/lambda-install` instead.
This could be a CMake quirk (which version are you using?)

**Comment by VikingExplorer** [comment]
cmake version 3.16.3
So, what do I need to change to make this work?

**Comment by marcomagdy** [comment]
I don't have a good solution for WSL2 (for reasons mentioned earlier).
I would try to either build on Linux (virtual) machine or build on a docker image of Ubuntu.

**Comment by VikingExplorer** [comment]
I'm just working on a POC. I've already uploaded a zip file for a Python lambda.  I've started the process to create a linux hyper-v machine, but it seems like intrusive overkill.  There must be a simpler way.  

All I need to to is build a single CPP file, link & create a zip file that includes the aws runtime. I'm looking through CMake, which seems like a giant ill-conceived preprocessor for make. I can't find the command to zip up files. I'd love to be able to run a few commands to build & zip.  It shouldn't be that hard.  Any help would be greatly appreciated.
